ecommerce/settings/devstack.py

```
@@ -23,25 +21,27 @@
 
 # PAYMENT PROCESSING
 PAYMENT_PROCESSOR_CONFIG = {
-    'cybersource': {
-        'soap_api_url': 'https://ics2wstest.ic3.com/commerce/1.x/transactionProcessor/CyberSourceTransaction_1.115.wsdl',
-        'merchant_id': 'fake-merchant-id',
-        'transaction_key': 'fake-transaction-key',
-        'profile_id': 'fake-profile-id',
-        'access_key': 'fake-access-key',
-        'secret_key': 'fake-secret-key',
-        'payment_page_url': 'https://testsecureacceptance.cybersource.com/pay',
-        'receipt_page_url': get_lms_url(settings.RECEIPT_PAGE_PATH),
-        'cancel_page_url': get_lms_url('/commerce/checkout/cancel/'),
-    },
-    'paypal': {
-        'mode': 'sandbox',
-        'client_id': 'fake-client-id',
-        'client_secret': 'fake-client-secret',
-        'receipt_url': get_lms_url(settings.RECEIPT_PAGE_PATH),
-        'cancel_url': get_lms_url('/commerce/checkout/cancel/'),
-        'error_url': get_lms_url('/commerce/checkout/error/'),
-    },
+    'default': {
```

Default is a misleading name here. I read default as: "If I don't override settings for a my microsite, it will use default settings". Outside test scope it is only used [here](https://github.com/edx/ecommerce/pull/630/files#r56902761) (I think), so possibly it should just go away.